services/QuillLMS/e2e/main.spec.ts

```diff
@@ -1,28 +1,43 @@
 import { test, expect } from '@playwright/test';
 
+function filterAndLogRequest(request, regex) {
+  const re = new RegExp(regex);
+  if (request.url().match(re)) {
+    console.log('>>', request.method(), request.url())
+  }
+}
+
+function filterAndLogResponse(response, regex) {
+  const re = new RegExp(regex);
+  if (response.url().match(re)) {
+    console.log('<<', response.status(), response.url())
+  }
+}
+
 test('has visible footer', async ({ page }) => {
-  await page.goto('http://localhost:5000');
+  await page.goto('/session/new');
 
-  await expect(page.locator('footer')).toBeVisible();
+  await expect(page.locator('p.sign-up-link')).toBeVisible();
 });
 
+test('@login form submission with valid credentials', async ({ page }) => {
+  page.on('request', request => filterAndLogRequest(request, 'login_through_ajax'));
```

Comment:
These three lines are not required for the test to pass, but I think it's helpful as a reference for how to deeply inspect a test that you're debugging.